db/schema.rb

```diff
@@ -66,6 +66,7 @@
     t.string "full_name"
     t.string "common_name"
     t.string "position"
+    t.string "picture"
```

Comment:
this isn't how you add fields to the db, you need to do a [migration](https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-a-migration)

Reply:
I've added       t.string :picture to the first migration file, is that right?